src/Analysis/Ast/Test/AnalysisTestBase.cs

```diff
@@ -58,13 +58,13 @@ public abstract class AnalysisTestBase {
         protected string GetAnalysisTestDataFilesPath() => TestData.GetPath(Path.Combine("TestData", "AstAnalysis"));
 
         protected Task<IServiceManager> CreateServicesAsync(InterpreterConfiguration configuration, IServiceManager sm = null)
-            => CreateServicesAsync(TestData.GetTestSpecificRootUri().AbsolutePath, configuration, sm);
+            => CreateServicesAsync(TestData.GetTestSpecificRootUri().AbsolutePath, configuration, null, sm);
 
-        protected async Task<IServiceManager> CreateServicesAsync(string root, InterpreterConfiguration configuration, IServiceManager sm = null) {
+        protected async Task<IServiceManager> CreateServicesAsync(string root, InterpreterConfiguration configuration, string stubCacheFolderPath = null, IServiceManager sm = null) {
             configuration = configuration ?? PythonVersions.LatestAvailable;
             configuration.AssertInstalled();
-            Trace.TraceInformation("Cache Path: " + configuration.DatabasePath);
-            configuration.DatabasePath = TestData.GetAstAnalysisCachePath(configuration.Version, true);
+            stubCacheFolderPath = stubCacheFolderPath ?? TestData.GetAstAnalysisCachePath(configuration.Version, true);
+            Trace.TraceInformation("Cache Path: " + stubCacheFolderPath);
```

Comment:
I know this is here for testing, but can the same thing get printed during a regular run? (like the interpreter path, search paths, etc)

Reply:
Ignore this, it's printed. I was running the wrong repo...